doc/tutorials/intermediate/structure_data_store.md

```diff
@@ -0,0 +1,330 @@
+# Structure with a DataStore
```

Comment:
I've migrated the `DataStore` into this ´intermediate` tutorial.

I've tried to structure it such that it can run with pyodide in this notebook. But also such that its a *real* example of an app that is structured across multiple files. We don't really have that today. And its one of the things that make Panel hard to use because users have to think about and invent their own structure.

The app lacks some finish. Visually as shown below. Use readable names, use the fill width, align better, format numbers etc. But **most important** when you change values in the widgets a lot of warnings and errors are raised in the terminal.

I hope that @philippjfr would fix the issues and maybe also polish a bit?

![image](https://github.com/holoviz/panel/assets/42288570/a9432f56-95dd-47a7-8c9e-cfc65b47d6d6)

```bash
param\reactive.py:991: UserWarning: Boolean Series key will be reindexed to 
match DataFrame index.
  obj = fn(obj, *resolved_args, **resolved_kwargs)
```

```bash
\pandas\core\indexing.py", line 2575, in check_bool_indexer
    raise IndexingError(
pandas.errors.IndexingError: Unalignable boolean Series provided as indexer (index of the boolean Series and of the indexed object do not match).
```

Maybe the use of `.rx` should be re-considered. Maybe .bind or .depends will make it more robust?